src/commands.js

```diff
@@ -43,6 +43,14 @@ module.exports = {
       })
     }
   },
+  share: { 
+    help: () => 'print a cabal key with you as admin. useful for sending to friends',
```

Comment:
Discussion point: do we want /share to share with you as admin, or with you + whoever is in your admin/mod-keys?

Reply:
@noffle good discussion point! my reasoning was as follows:

what i reckoned was that it would get so absurdly long as to be useless if we shared all the admin + modkeys, which is why i opted for sharing oneself as an admin; reasoning that would have a similar effect 

(at least that was what i was thinking at the time; given that the passed-in admin/modkeys don't actually issue admin/mods as statements then they won't be transitively be available)

Reply:
Hey! I wasn't sure where to continue this convo :)

As I understand it, most people have one admin key. So a /share that used your admin keys should also just include *one* key, right? It feels like it might get confusing if inviting another user gives them a different moderation view than yours.